### PR TITLE
Properties docs update

### PR DESCRIPTION
**Pull request by gcwiak**

### CODE REVIEW

File: core/src/main/java/io/nixer/nixerplugin/core/detection/config/FailedLoginRatioProperties.java [hunk 1]
```diff
@@ -17,32 +17,12 @@
...
-     * <br/>
-     * <code>
-     *     failed-login-ratio = (100 * number or failed logins) / (number of all logins)
-     * </code>
-     * <br/>
-     * <br/>
-     *
-     * Activation and deactivation levels create hysteresis to better cope with credential stuffing and also to prevent too frequent
-     * activation/deactivation events.
+     * Percentage value of failed-login-ratio metric above which an activation event will be generated.
```
Comment (j-bron) [inline core/src/main/java/io/nixer/nixerplugin/core/detection/config/FailedLoginRatioProperties.java:20]: Why this change? It would be good to have this in generated properties from script...

Comment (gcwiak) [reply core/src/main/java/io/nixer/nixerplugin/core/detection/config/FailedLoginRatioProperties.java:20]: As this is quite a low level implementation detail I think it would be better to keep it on the implementation itself, so I moved it to the FailedLoginRatioRule javadocs. 
Having it separately, on the properties, is prone to outdating when the implementation changes. Also bloating the properties documentation with such details would make the docs structure chaotic and not easy to follow - instead I'd rather kept the properties descriptions concise and created a dedicated section about the rules.

Comment (j-bron) [reply core/src/main/java/io/nixer/nixerplugin/core/detection/config/FailedLoginRatioProperties.java:20]: It is actually quite important parameters that together with minimal sample size impact greatly on system response during credential stuffing.
It will be outdated anyway no matter when we would put it.

I preferred previous layout to be honest, properties can be viewed by more people theoretically, documenting on the class makes less sense as developers can just read the code.

File: core/src/main/java/io/nixer/nixerplugin/core/detection/rules/ratio/FailedLoginRatioRule.java [hunk 2]
```diff
@@ -11,6 +11,26 @@
 import org.apache.commons.logging.LogFactory;
 import org.springframework.util.Assert;
 
+/**
```
Comment (j-bron) [inline core/src/main/java/io/nixer/nixerplugin/core/detection/rules/ratio/FailedLoginRatioRule.java:14]: This description is too detailed, should be on the method level. It does not describe minimumSampleSize and at this level of detail, it should describe all logic. 
I would not document in detail these as one can just read the code.

File: core/src/main/java/io/nixer/nixerplugin/core/detection/config/FailedLoginRatioProperties.java (hunk 1 above)
Comment (gcwiak) [reply core/src/main/java/io/nixer/nixerplugin/core/detection/config/FailedLoginRatioProperties.java]: Ok, I see your point here. I restored the previous layout with some improvements and made the rule docs less detailed instead.

File: core/src/main/java/io/nixer/nixerplugin/core/detection/rules/ratio/FailedLoginRatioRule.java (hunk 2 above)
Comment (gcwiak) [reply core/src/main/java/io/nixer/nixerplugin/core/detection/rules/ratio/FailedLoginRatioRule.java:14]: Ok, I left here just a brief description.
